### DISCUSSION TITLE
Day 2 actions and workflows

### DISCUSSION THREAD

**Discussion by Shnitselon** [QualiTorque/qualitorque.github.io#245] [Announcements]
We're happy to announce the release of Actions and Workflows.

Actions are a set of built-in tasks you can run on specific environment VM resources, and include Power on, Power off, Restart and Connect, while Workflows are admin-defined actions that run on all VMs in specific spaces or across the entire Torque account.

For details, see [Workflows](https://docs.qtorque.io/admin-guide/governance/workflows) and [Run Day 2 Actions on Your Environment](https://docs.qtorque.io/getting-started/Run%20day2%20actions).

**Comment by cbowldstsi** [comment]
Are custom Actions supported, or planned? For example, take a snapshot of a VM. Or run an Ansible or shell script.

**Comment by shirabendor-quali** [comment]
Hi @cbowldstsi ,
Yes, custom actions are definitely planned and are on the roadmap.
Stay tuned...
